CHANGELOG.md

```diff
@@ -1,33 +0,0 @@
-### 0.1.2 (2015-11-25)
```

Comment:
The gcloud module works now but not very good with travis. Every project that wants to use this module has to add special travis option (see 2ad2bb0)